add-topic-to-rst.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+#
+#    Copyright 2022 OICR and UCSC
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# This script determines if a specified RST references a discourse topic, and if not, it attempts to
+# extract some information from the RST, create a Discourse topic, and modify the RST to reference the new topic.
+
+set -e
+set -u
+
+DOCS_URL="https://docs.dockstore.org/en/develop"
```

Comment:
Our official documentation is on `stable`, not `develop` (used to be `develop`, which is where you probably got this from), although I don't know the implications -- back links are wrong?

Reply:
https://ucsc-gi.slack.com/archives/C02A2L5LHD1/p1670974532857679